about.html

```diff
@@ -0,0 +1,130 @@
+                <a href="index.html" class="web-nav-link">Home</a>
+                <a href="#program" class="web-nav-link">Program</a>
+                <a href="#devs" class="web-nav-link">Developers</a>
+                <a href="#partner" class="web-nav-link">Sponsor</a>
+                <a href="#" class="web-nav-link ">CN Campaign</a>
+            </div>
+        </div>
+    </header>
+    <br><br><br>
+    <section class="content">
+        <h1 class="about-title">
+            Microverse Developers Global Conference 2022
+        </h1>
+        <p class="about-description">
+            The Microverse Developers Global Conference brings together
+            the community of Software Developers all over the world. This year,
+            we bring you some of the best Developers from around the world.
+        </p>
+        <p class="about-mail">
+            Please contact us for any further questions about Microverse Developers Global Conference 2022!
+        </p>
+        <h5 class="about-venue">
+            MicroverseDGC2022@yahoo.com
+        </h5>
+    </section>
+
+    <section class="logo">
+        <h1 class="logo-title">Microverse Developers Global Conference 2022 logo</h1>
+        <hr>
+        <p>
+            The Microverse Developers Global Conference 2022 logo was decided
+            through the logo competition from March 14th to March 15th 2022
+        </p>
+        <div class="cn-logo">
+            <img src="images/logo2.png" alt="logo">
+        </div>
+
+    </section>
+
+    <section class="past">
+        <h1 class="past-title">Microverse Developers Global Conference 2022 logo</h1>
+        <hr>
+        <p>
+            Take a look at the past Microverse Developers Global Conference 2022 which took place in Nigeria and in
+            South Africa.
+        </p>
+        <div class="event">
+            <div class="summit">
+                <div class="center">
+                    <p>2020</p>
+                    Microverse Developers Global Conference 2020 in Nigeria
+                </div>
+            </div>
+            <div class="summit">
+                <div class="center">
+                    <p>2021</p>
+                    Microverse Developers Global Conference 2021 in South Africa
+                </div>
+            </div>
+        </div>
+
+    </section>
+
+    <section id="partner">
+        <div>
+            <h1 class="partners-title">Partners</h1>
+            <hr>
+        </div>
+        <div class="partners">
+            <img src="images/microverse.png" alt="Microverse">
+            <img src="images/microsoft.svg" alt="Microsoft">
+            <img src="images/bic.png" alt="Bootcamp">
+        </div>
+    </section>
+
+    <footer>
+        <div class="foot">
+            <a href="index.html"><img src="./images/logo.png" class="imgfoot" alt="logo"></a>
+        </div>
+        <p> Developers Conference 2022. Some rights reserved.</p>
+    </footer>
```

Comment:
Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_11](https://user-images.githubusercontent.com/69638013/162840676-d5f8287c-dfa7-4d85-9f06-81635e93a77d.png)  | ![Screenshot_12](https://user-images.githubusercontent.com/69638013/162840634-d4027124-68e7-46e5-8ea1-5eab4e65d3c8.png)

- [ ] Kindly add the right color to your footer section, check screenshot abode

Reply:
Implemented 👍🏽